package.json

```diff
@@ -10,7 +10,7 @@
     "premocha": "npm run lib",
     "mocha": "mocha tests/*.ts tests/**/*.ts --require ts-node/register",
     "postmocha": "rm -rf tests/**/*.js",
-    "test": "npm run lint && npm run mocha && npm run doctest",
+    "test": "npm run lint && npm run mocha",
```

Comment:
Wait nevermind, I see now that a bunch of stuff isn't implemented, that makes sense.